collect_app/src/main/java/org/odk/collect/android/activities/FormFillingActivity.java

```diff
@@ -877,7 +877,7 @@ protected void onActivityResult(int requestCode, int resultCode, final Intent in
         // button or another question to jump to so we need to rebuild the view.
         if (requestCode == RequestCodes.HIERARCHY_ACTIVITY || requestCode == RequestCodes.CHANGE_SETTINGS) {
             activityDisplayed();
-            formEntryViewModel.refreshSync();
```

Comment:
Instead of removing `refreshSync` could we keep it and `refresh` and just switch to using `refreshSync` everywhere (and `updateIndex(false)` inside `FormEntryViewModel`. We definitely want to move away from loading select choices and other things that happen when moving screens on the UI thread, so even if it's unused I'd like to leave the "infrastructure" in. Having `refreshSync` deprecated also flags that we don't want to add anything else in that uses it.

Reply:
Done.